src/main/java/org/jenkinsci/plugins/GithubSecurityRealm.java

```diff
@@ -636,12 +637,20 @@ public UserDetails loadUserByUsername(String username)
             throws UsernameNotFoundException, DataAccessException {
```

Comment:
not really needed, `UserMayOrMayNotExistException` is a `UsernameNotFoundException`